latest/models/image.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import annotations
+from typing import Literal, Optional
+from pydantic import BaseModel, ConfigDict, conlist
+from typing import Annotated, Hashable, List, TypeVar
+from pydantic_core import PydanticCustomError
+from pydantic import AfterValidator, Field, ValidationError
+import json
+
+T = TypeVar('T', bound=Hashable)
+
+def _validate_unique_list(v: list[T]) -> list[T]:
+    if len(v) != len(set(v)):
+        raise PydanticCustomError('unique_list', 'List must be unique')
+    return v
+
+UniqueList = Annotated[List[T], AfterValidator(_validate_unique_list), Field(json_schema_extra={'uniqueItems': True}, min_length=1)]
+
+class Axis(BaseModel, frozen=True):
+    name: str
+    type: Optional[str] = None
+    unit: Optional[str] = None
+
+class ScaleTransform(BaseModel, frozen=True):
+    type: Literal["scale"]
+    scale: conlist(float, min_length=1)
+
+class TranslationTransform(BaseModel, frozen=True):
+    type: Literal["translation"]
+    translation: conlist(float, min_length=1)
+
+class Dataset(BaseModel, frozen=True):
+    path: str
+    coordinateTransformations: tuple[ScaleTransform] | tuple[ScaleTransform, TranslationTransform]
+
+class Multiscale(BaseModel, frozen=True):
+    """
+    The multiscale datasets for this image
+    """
+    name: Optional[str] = None
+    datasets: conlist(Dataset, min_length=1)
+    axes: UniqueList[Axis]
+    coordinateTransformations: Optional[tuple[ScaleTransform] | tuple[ScaleTransform, TranslationTransform]] = None
```

Comment:
Is a pydantic tuple equivalent to list in JSON?

Reply:
JSON arrays are equivalent to python lists, but the spec defines that coordinateTransformations is typed collection with fixed length, so on the python side it's a union of tuples.